README.md

```diff
@@ -1,4 +1,7 @@
-# Zeek Package Template (Beta!)
+# Zeek Package Template
+
+[![Zeek matrix tests](https://github.com/zeek/package-template/actions/workflows/zeek-matrix.yml/badge.svg)](https://github.com/zeek/package-template/actions/workflows/zeek-matrix.yml)
+[![Zeek nightly tests](https://github.com/zeek/package-template/actions/workflows/zeek-nightly.yml/badge.svg)](https://github.com/zeek/package-template/actions/workflows/zeek-nightly.yml)
```

Comment:
Very true. Btw — the biggest downside I see with these nightly jobs is that they die after 60(?) days unless you re-confirm them. It'd be really cool if the email notifications would flag this as that time comes closer. I think there's a one-shot notification from Github but I manage to miss that regularly. :-/

Reply:
Okay so this is trickier than I thought. The ci-email-action we use for our CI notifications works on the notion of check suites, which seems to be something that only Github apps (like Cirrus) provide. I never get a `check_suite` trigger on my workflow runs. Let me know if I'm missing anything: https://github.com/ckreibich/package-template

There are recipes out there like [this one](https://stackoverflow.com/a/67265461) that look more natural for what we want, but I'd rather not pull in yet another 3rd-party action and instead upgrade our own. So I suggest we make this follow-up work.

In the meantime there's also the fact that Github does send a notification — it just doesn't go to the whole team, so that's certainly not ideal.

Reply:
Can you point me to the email action you used? I do see lots of check suites for the latest commit on your fork ckreibich/package-template@36c6a51b93d0262e11addd0527942e64412f4cce (there are many since you use an aggressive cron schedule, so the same SHA sees lots of test runs), see the https://api.github.com/repos/ckreibich/package-template/commits/master/check-runs.

Just guessing, I wonder whether the problems you are seeing instead have something to do with permissions or an action just being skipped in forks.

Reply:
I used `zeek/ci-email-action@master`. I believe the exact workflow was this, but I no longer have the branch around:
```
name: CI Email Notification
on:
  check_suite:
    types: [completed]
jobs:
  notify:
    runs-on: ubuntu-latest
    steps:
      - name: Show the event
        run: cat "$GITHUB_EVENT_PATH" || true

      - name: Send CI Email Notification
        uses: zeek/ci-email-action@master
        env:
          CI_APP_NAME: "Cirrus CI"
          BRANCH_REGEX: "master|release/.*"
          SKIP_CONCLUSIONS: 'cancelled,neutral,skipped'
          SMTP_HOST: ${{ secrets.SMTP_HOST }}
          SMTP_PORT: ${{ secrets.SMTP_PORT }}
          SMTP_USER: ${{ secrets.SMTP_USER }}
          SMTP_PASS: ${{ secrets.SMTP_PASS }}
          MAIL_FROM: ${{ secrets.MAIL_FROM }}
          MAIL_TO: ${{ secrets.MAIL_TO }}
          MAIL_REPLY_TO: ${{ secrets.MAIL_REPLY_TO }}
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
So basically what we have in Zeek, without the repo filter.